src/main/java/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView.java

```diff
@@ -606,11 +606,15 @@ private int triggerBuild(final AbstractProject<?, ?> triggerProject, final Abstr
         if (manualTrigger != null) {
             final Set<String> downstreamProjectsNames =
                     Sets.newHashSet(Splitter.on(",").trimResults().split(manualTrigger.getDownstreamProjectNames()));
-            if (downstreamProjectsNames.contains(project.getFullName())) {

```

Comment:
That might be possible ;-) 

Today, I don't have the time, but I will give a new try and let you know it early next week. I think that even tried what you say, but this was so many time ago that now I cannot longer remember it.